pkg/cmd/reset.go

```diff
@@ -17,17 +17,19 @@ limitations under the License.
 package cmd
 
 import (
+	"fmt"
+
 	"github.com/MakeNowJust/heredoc/v2"
 	"github.com/pkg/errors"
 	"github.com/spf13/cobra"
 	"github.com/spf13/pflag"
-
```

Comment:
This new line should still stay there because we put `k8c.io` imports in a dedicated group.

Reply:
Sure

Reply:
This should be solved